### ISSUE TITLE
[Suggestion] Adding Traits

### ISSUE THREAD

**Issue by mxnmnm** [RCXcrafter/TweakersConstruct#22]
First let's start off with the bug which is some traits do not like being added those being crude, writable, and splinters.  I'm sure there are some modded ones but I haven't checked yet.

Now to the request which would save me a good amount of time.  Can we please get an option to add traits to specific things without it deleting the other traits?  Like for example I'm trying to change up attack damage on all materials and this resulted in bows also doing more damage so I decided to make a contenttweaker trait that reduces damage and I want to put it on all bows but now I can't do that without overwriting the 100+ items which i was gonna do but I figured I might as well make the request wait for you to implement it/you telling me your not gonna add it so i can do this grindy stuff with peace of mind that it was the only way.

tl;dr- some traits no worky and adding different traits to tools is too grindy

**Comment by mxnmnm** [comment]
So apparently I had to put a number in front of some of the traits in order to get them to work and it seems to be working for most of them although I am having trouble with some of them but I'm sure I can figure that out because it's probs some compat issues

**Comment by mxnmnm** [comment]
So simply having a way to add traits or at least not having an entire material's traits removed when you try to add one extra trait would be nice.  Thanks!

**Comment by RCXcrafter** [comment]
In version 1.5.2 you'll be able to add ":false" after a line to keep all the existing traits. Sorry for taking so long, I hope you haven't done it the grindy way yet.
